lib/miq_automation_engine/engine/miq_ae_engine.rb

```diff
@@ -69,6 +69,19 @@ def self.deliver_queue(args, options = {})
 
   def self.deliver(*args)
     options     = options_from_args(args)
+
+    # HACK: If we are launching a specific Automate instance representing a workflow, then we run that instead
+    if options[:namespace] == "System" && options[:class_name] == "Workflow"
+      workflow_id = options[:instance_name].split("-").last.to_i
+      workflow = Workflow.find(workflow_id)
+
+      task_id = workflow.execute(inputs: options)
+      MiqTask.wait_for_taskid(task_id)
```

Comment:
this wait_for_taskid makes me a little nervous here.
But if we need this temporarily then 👍